tests/unit/createPackage/models/tasksModel.spec.ts

```diff
@@ -171,7 +172,7 @@ describe('TasksManager', () => {
       getFileSizeSpy.mockResolvedValue(2000);
       const expirationTime = new Date();
       const expectedCallbackData: ICallbackDataBase = {
-        fileUri: 'http://download-service/downloads/test/test.gpkg',
+        fileUri: 'http://download-service/downloads/test' + sep + 'test.gpkg',
```

Comment:
`${gpkgsLocation}${separator}${packageDirectoryName}${separator}${packageName}`;
concat it like this convention to save consistens